src/ipc-types.d.ts

```diff
@@ -45,6 +45,12 @@ interface StartedSiteDetails extends StoppedSiteDetails {
 
 type SiteDetails = StartedSiteDetails | StoppedSiteDetails;
 
+type NewSiteDetails = {
+	id: string;
+	path: string;
+	name: string;
+};
```

Comment:
```suggestion
type NewSiteDetails = Pick< SiteDetails, 'id' | 'path' | 'name' >;
```
Nit, but this allows us to continue using a canonical type definition for the sites objects.